lib/models/addons-factory.js

```diff
@@ -35,9 +35,15 @@ class AddonsFactory {
     logger.info('     addon names are:', Object.keys(addonPackages));
 
     for (let name in addonPackages) {
+      console.log(`### factory.initAddons for '${name}'`);
```

Comment:
before merging, these logs will have to be removed, or switched to use heimdall-logger/debug

Reply:
Re this one, I was going to remove it. Will do so for the next checkin (it was just there for debugging).